tutorials/nservicebus-sagas/2-timeouts/Solution/Sales/BuyersRemorsePolicy.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace Core_7.BuyersRemorseTimeoutRequest
+{
+    using NServiceBus;
+    using NServiceBus.Logging;
+    using System;
+    using System.Threading.Tasks;
+    using Messages;
+
+    class BuyersRemorsePolicy
+        : Saga<BuyersRemorseState>
```

Comment:
Again, additional line.

Reply:
Even if I agree this is in the final solution, I'd rather keep it consistent with lesson 1. Thoughts?